GiniVision/Classes/SupportedFormatsViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SupportedTypesViewController.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/19/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+final class SupportedFormatsViewController: UITableViewController {
+    
+    let reuseIdentifier = "reuseIdentifier"
```

Comment:
"reuseIdentifier"? That's a bit ambiguous. Why not something like "SupportedFormatsCellIdentifier"?

Reply:
Yes, I forgot to change it after Xcode created it. 👍